### DISCUSSION TITLE
Connecting servomotor

### DISCUSSION THREAD

**Discussion by FannyCazettes** [harp-tech/protocol#81] [Q&A] [answered]
How do you connect the servomotor to the behavior board? We have used both D02 and D03, and configured one as the signal and one as the power, but we are not sure this is correct.

**Comment by artursilva0** [comment, marked as answer]
You can connect the signal of the servomotor to DO2 or DO3, the power you should connect to the +5V screw terminal and the ground to the GND screw terminal. 

I am assuming that the servomotor requires less than ~200-300mA. If this is not the case, then you should use an external supply, connect the ground of the external supply to the GND screw terminal and use the DO2 or DO3 as the signal.

**Reply by FannyCazettes** [reply]
Thanks, that helps. My motor requires more than 300mA, so we will need an external supply.
